src/Bridge/Doctrine/Orm/State/UriVariablesHandlerTrait.php

```diff
@@ -38,35 +38,19 @@ private function handleUriVariables(QueryBuilder $queryBuilder, array $identifie
                 $propertyIdentifier = $uriVariable->getIdentifiers()[0];
                 $joinAlias = $queryNameGenerator->generateJoinAlias($inverseProperty);
 
-                $queryBuilder->join(
-                    $uriVariable->getTargetClass(),
-                    $joinAlias,
-                    'with',
-                    "$alias.$propertyIdentifier = $joinAlias.$inverseProperty"
-                );
+                $queryBuilder->join($uriVariable->getTargetClass(), $joinAlias, 'with', "$alias.$propertyIdentifier = $joinAlias.$inverseProperty");
 
-                $expression = $queryBuilder->expr()->eq(
-                    "{$joinAlias}.{$propertyIdentifier}",
-                    $placeholder
-                );
+                $expression = $queryBuilder->expr()->eq("{$joinAlias}.{$propertyIdentifier}", $placeholder);
             } elseif ($property = $uriVariable->getProperty()) {
                 $propertyIdentifier = $uriVariable->getIdentifiers()[0];
                 $joinAlias = $queryNameGenerator->generateJoinAlias($property);
 
-                $queryBuilder->join(
-                    "$alias.$property",
-                    $joinAlias,
-                );
+                $queryBuilder->join("$alias.$property", $joinAlias, );
```

Comment:
extra comma (probably all these formatting changes should be reverted) 

```suggestion
                $queryBuilder->join("$alias.$property", $joinAlias);
```

Reply:
As there are not intended for this PR, I have reverted all code style changes.
Thank you.